handbook/brand.md

```diff
@@ -91,7 +91,7 @@ For draft articles, please refer to [Publishing articles on fleetdm.com.](https:
 
 For making edits to published articles:
 
-1. Log in to Medium via [fleetdm.com/blog](https://fleetdm.com/blog).
+1. Log in to Medium via [the fleet blog](https://fleetdm.com/blog).
```

Comment:
```suggestion
1. Log in to [Medium](https://fleetdm.com/blog).
```

Reply:
Good shout cleaning up these links. In this case, though, including the link naturally in the sentence changes the context. Instead, let's change it to "Log in to Medium."